brultech_serial2mqtt/config/config_mqtt.py

```diff
@@ -261,6 +205,41 @@ def status_topic(self, device_serial: int) -> str:
         """Return the topic used to convey service status (via birth and will)."""
         return f"{self.topic_prefix(device_serial)}/status"
 
+    @property
+    def tls_params(self) -> Optional[MqttTlsParams]:
+        """Return TLS parameters"""
+
+        if not self._usetls:
+            # Return nothing if not using TLS
+            return None
+
+        def cert_reqs(cert_req: bool) -> ssl.VerifyMode:
+            if not cert_req:
+                return ssl.CERT_NONE
+            else:
```

Comment:
minor nit: I'm not a fan of else-after-return.  Can you just remove the `else:` here?